src/test/hvm.rs

```diff
@@ -440,7 +443,7 @@ proptest! {
 #[case(&format!("(>> #{} #119)", U120::MAX), 1)]
 #[case(&format!("(>> #{} #118)", U120::MAX), 3)]
 #[case(&format!("(+ #{} #1)", U120::MAX), *U120::ZERO)]
-#[case("(- #0 #1)", *U120::MAX)]
```

Comment:
I think you meant to add a new case and not erase this one

Reply:
Oh! This case was not deleted, just moved it upwards in order to make it a little bit more aligned.